pyflakes/checker.py

```diff
@@ -1160,9 +1166,11 @@ def addArgs(arglist):
                 if arg in args[:idx]:
                     self.report(messages.DuplicateArgument, node, arg)
 
-        for child in annotations + defaults:
-            if child:
```

Comment:
I think this was redundant. The check was added in https://github.com/PyCQA/pyflakes/commit/34fd8ba800d288a41bde4c77d6be61effe94e177#diff-f0429ba5bb9de6e83abcf687aca4c012R751, but `handleNode` already checks for `node is None`, and I don't see how `child` can be a falsy value other than `None` here.

The previous revision did not have this extra falsiness check on `defaults`: https://github.com/PyCQA/pyflakes/commit/34fd8ba800d288a41bde4c77d6be61effe94e177#diff-f0429ba5bb9de6e83abcf687aca4c012L750.